src/App.js

```diff
@@ -1,14 +1,21 @@
 import './App.css';
-import VehicleList from './VehicleList';
-import TrafficLight from './TrafficLight';
+import { useState } from 'react';
+import VehicleList from './Components/VehicleList';
+import TrafficLight from './Components/TrafficLight';
 
 function App() {
   // track the following state with a few useState hooks:
   // lightColor should be a string that starts out as 'red'
   // lizardSize should be a number that starts out as 10
   // alienSize should be a number that starts out as 10
   // traffic is complicated. It should be an array of strings that starts out as ['car', 'truck']
+  const [lightColor, setLightColor] = useState('red');
+  // const [lizardSize, setLizardSize] = useState(10);
+  // const [alienSize, setAlienSize] = useState(10);
+  // const [traffic, setTraffic] = useState(['car', 'truck']);
```

Comment:
Looks like you have all your state set up to track, just need to utilize it and pass the appropriate props to your various components.